storybook/src/docs/developer-guide/using-links-with-routers.docs.mdx

```diff
@@ -0,0 +1,56 @@
+{/* @license CC0-1.0 */}
+
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/Developer Guide/Using links with routers" />
```

Comment:
Is this just as clear? Otherwise, this title is so much longer than the others in the table of contents…


```suggestion
<Meta title="Docs/Developer Guide/Links and routing" />
```

Reply:
4 words is not too bad, right? 'Links and routing' is very non-descriptive imo. Or do you think people looking for this information will find it with that title?

Reply:
But ‘Using’ doesn’t add much, does it? We could prefix everything with that word. It’s a bit out of character like this. I’d even still like ‘Routing’ best. Quite petty, I know :)

![Developer Guide toc](https://github.com/user-attachments/assets/49b7f582-53bc-46bd-9d6c-f27dae36c1fc)

Reply:
As discussed, I've renamed this to 'Routing libraries'